docs/build.js

```diff
@@ -24,6 +25,8 @@ async function init() {
   console.log('Cleaning up output directory ' + outputDir);
   await rm(outputDir, { force: true, recursive: true });
   await mkdir(outputDir);
+  console.log(`Copying file ${join(cwd, 'LICENSE.md')}`);
+  console.log(`          to ${join(inputDir, 'LICENSE.md')}`);
   await copyFile(join(cwd, 'LICENSE.md'), join(inputDir, 'LICENSE.md'));
```

Comment:
Let's add something like:

```
await copyFile(join(cwd, 'marked.min.js'), join(outputDir, 'marked.min.js''));
```

Then in the client side JS when you select latest, you can `fetch(new URL('/marked.min.js', window.location))`.

Reply:
Is there a way to get the PR vercel link with just a PR#?

Reply:
I found it based on the vercel comment View Preview link

Reply:
[PR Demo](https://marked-website-git-fork-uzitech-remove-built-markedjs.vercel.app/demo/?text=%23%20test&options=%7B%0A%20%22async%22%3A%20false%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22extensions%22%3A%20null%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22hooks%22%3A%20null%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%0A%7D&version=pr%3A2963)

Reply:
or we don't even need the pr version because now we can go to the preview and use /marked.min.js

Reply:
Exactly, we don’t need an input anymore. Either you select an old version or you select “HEAD”. And HEAD for a production deployment is master branch and for a preview deployment it’s the PR branch.